dipy/viz/colormap.py

```diff
@@ -255,3 +261,52 @@ def line_colors(streamlines, cmap='rgb_standard'):
                     for streamline in streamlines]
 
     return np.vstack(col_list)
+
+
+lowercase_cm_name = {'blues': 'Blues', 'accent': 'Accent'}
+
+
+def create_colormap(v, name='jet', auto=True):
```

Comment:
I didn't say one colormap is good for all the different jobs you might want to do with it, just that this one is particularly bad for any task, and we shouldn't use it as the default (see: https://jakevdp.github.io/blog/2014/10/16/how-bad-is-your-colormap/). Some visualization software goes so far as to prevent users from using "jet": https://github.com/mwaskom/seaborn/commit/9f5a177639a7b5d5d8c7cce98def0e2e10c947b0

I don't think we need to go that far, but users will tend to stick to the default, so I think we should change it.